test/contracts/consumer/fabric8auth/consumer_test.go

```diff
@@ -0,0 +1,69 @@
+package fabric8auth
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-auth/test/contracts/model"
+	"github.com/fabric8-services/fabric8-wit/test/contracts"
+	"github.com/pact-foundation/pact-go/dsl"
+	"github.com/pact-foundation/pact-go/types"
+)
+
+// TestWitAPIConsumer runs all user related tests
+func TestWitAPIConsumer(t *testing.T) {
+	log.SetOutput(os.Stdout)
+
+	var pactDir = os.Getenv("PACT_DIR")
+	var pactVersion = os.Getenv("PACT_VERSION")
+
+	var pactConsumer = "Fabric8Wit"
+	var pactProvider = "Fabric8Auth"
+
+	var pactBrokerURL = os.Getenv("PACT_BROKER_URL")
+	var pactBrokerUsername = os.Getenv("PACT_BROKER_USERNAME")
+	var pactBrokerPassword = os.Getenv("PACT_BROKER_PASSWORD")
+
+	// Create Pact connecting to local Daemon
+	pact := &dsl.Pact{
+		Consumer:             pactConsumer,
+		Provider:             pactProvider,
+		PactDir:              pactDir,
+		Host:                 "localhost",
+		LogLevel:             "INFO",
+		PactFileWriteMode:    "overwrite",
+		SpecificationVersion: 2,
+	}
+	defer pact.Teardown()
+
+	// Test interactions
+	AuthAPIStatus(t, pact)
```

Comment:
It would be very helpful if you could use sub-tests for all the different contract tests.
```suggestion
	t.Run("api status", func(t *testing.T){ AuthAPIStatus(t, pact) })
```

Reply:
Thanks for the tip - fixed.